### DISCUSSION TITLE
xcode to github

### DISCUSSION THREAD

**Discussion by StephenPowerRMIT** [community/community#22617] [New to GitHub]
<p>I am a online student studying at moment: App building, we have a small group, we must use xcode and store in our Github repositiry, I can’t work out how to bring it from Github to Xcode or from Xcode to Github for the project I decided to start myself. Please help me!!</p>

**Comment by ghost** [comment, marked as answer]
<p>Hey <a class="mention" href="/stephenpowerrmit">@stephenpowerrmit</a>, help’s arrived!</p>
<p>Welcome to the GitHub Community 🎉</p>
<p>It sounds like you might be interested in this Medium story by XCTEQ: <a href="https://medium.com/xcblog/the-marriage-of-github-and-xcode-9-at-wwdc-821b78eb6797" rel="nofollow noopener">https://medium.com/xcblog/the-marriage-of-github-and-xcode-9-at-wwdc-821b78eb6797</a> - it’ll show you how to set up and use GitHub in XCode.</p>
<p>Otherwise, you can also use the Git CLI (“Commend Line Interface”) with the folder the XCode files are in - check out the GitHub Labs (free) courses on Git and GitHub here: <a href="https://lab.github.com/courses?tag=GitHub,Git" rel="nofollow noopener">https://lab.github.com/courses?tag=GitHub,Git</a>, and get started with Git here: <a href="https://git-scm.com" rel="nofollow noopener">https://git-scm.com</a></p>
<p>Let me know how you go!</p>
